### ISSUE TITLE
Extern font

### ISSUE THREAD

**Issue by richelbilderbeek** [iloverichel/iloverichel.github.io#1]
Vraag van @jorn600: 

```
Ik kan de externe font niet op iloverichel.github.io zetten
```

Reactie van @richelbilderbeek:

Wat bedoel je? Is 'Upload file' vanuit GitHub niet voldoende?

**Comment by JornPrengerFormeel** [comment]
nee ik snap niet hoe ik hem op de site moet zetten

**Comment by richelbilderbeek** [comment]
Ik zie in de HTML code dit soort dingen:

```
../iloverichel.github.io/ka1[1].ttf
```

Ik zou voorstellen:
- [ ] upload het font met een simpele naam, zoals `iloverichel.ttf`
- [ ] probeer eens: vervang `../iloverichel.github.io/iloverichel.ttf` door `iloverichel.ttf`

Succes!

**Comment by JornPrengerFormeel** [comment]
Snap jij hoe ik het moet oplossen

**Comment by richelbilderbeek** [comment]
Ja, zoals ik hierboven heb beschreven. Probeer eerst zelf :sunrise:

**Comment by richelbilderbeek** [comment]
En, al gelukt?
